src/business/meals.js

```diff
@@ -0,0 +1,38 @@
+function compareTimesUsed(mealA, mealB) {
```

Comment:
this was a simple way of keeping business logic separate from views. this let me not only reuse these functions across components, but across both versions of the app with no modification whatsoever